literature.bib

```diff
@@ -3277,6 +3282,109 @@ @inproceedings{SL:VaMoS21
+	title     = {{The Sandwich Principle: Assessing the Didactic Effect in Lectures on ``Cleft Lips and Palates''}},
+	journal   = BMCME,
+	publisher = Springer,
+	volume    = 20,
+	pages     = {1--7},
+	year      = 2020
+}
+
+@inproceedings{BRS:GPCE20,
+	author    = {Tabea Bordis and Tobias Runge and Ina Schaefer},
+	title     = {{Correctness-by-Construction for Feature-Oriented Software Product Lines}},
+	booktitle = GPCE,
+	publisher = ACM,
+	pages     = {22--34},
+	doi       = {10.1145/3425898.3426959},
+	year      = 2020
+}
+
+@inproceedings{AKM:SPLC20,
+	author    = {Assun\c{c}\~{a}o, Wesley K. G. and Kr\"{u}ger, Jacob and Mendon\c{c}a, Willian D. F.},
+	title     = {{Variability Management Meets Microservices: Six Challenges of Re-Engineering Microservice-Based Webshops}},
+	booktitle = SPLC,
+	publisher = ACM,
+	address   = NY,
+	isbn      = 9781450375696,
+	doi       = {10.1145/3382025.3414942},
+	year      = 2020,
+	articleno = 22
+}
+
+@inproceedings{DLP:ISoLA20,
+	author    = {Ferruccio Damiani and Michael Lienhardt and Luca Paolini},
+	title     = {{On Slicing Software Product Line Signatures}},
+	booktitle = ISoLA,
+	publisher = Springer,
+	pages     = {81--102},
+	doi       = {10.1007/978-3-030-61362-4\_5},
+	year      = 2020
+}
+
+@inproceedings{GBKR:SPLC20,
+	author    = {Gr{\"u}ner, Sten and Burger, Andreas and Kantonen, Tuomas and R{\"u}ckert, Julius},
+	title     = {{Incremental Migration to Software Product Line Engineering}},
+	booktitle = SPLC,
+	publisher = ACM,
+	pages     = {1--11},
+	doi       = {10.1145/3382025.3414956},
+	year      = 2020
+}
+
+@article{GM:EMSE20,
+	author    = {Mario Gleirscher and Diego Marmsoler},
+	title     = {{Formal Methods in Dependable Systems Engineering: A Survey of Professionals From Europe and North America}},
+	journal   = EMSE,
+	publisher = Springer,
+	volume    = 25,
+	number    = 6,
+	pages     = {4473--4546},
+	doi       = {10.1007/s10664-020-09836-5},
+	year      = 2020
+}
+
+@incollection{GtBvdP:20,
+	author    = {Hubert Garavel and Maurice H. ter Beek and Jaco van de Pol},
+	title     = {{The 2020 Expert Survey on Formal Methods}},
+	booktitle = FMICS,
+	publisher = Springer,
+	pages     = {3--69},
+	doi       = {10.1007/978-3-030-58298-2\_1},
+	year      = 2020
+}
+
+@inproceedings{KMB:SPLC20,
+	author    = {Kr{\"u}ger, Jacob and Mahmood, Wardah and Berger, Thorsten},
+	title     = {{Promote-pl: A Round-Trip Engineering Process Model for Adopting and Evolving Product Lines}},
+	booktitle = SPLC,
+	publisher = ACM,
+	pages     = {1--12},
+	doi       = {10.1145/3382025.3414970},
+	year      = 2020
+}
+
+@inproceedings{S:APLAS20,
+	author    = {Steinh\"{o}fel, Dominic},
+	title     = {{REFINITY to Model and Prove Program Transformation Rules}},
+	booktitle = APLAS,
+	publisher = Springer,
+	pages     = {311--319},
+	doi       = {10.1007/978-3-030-64437-6\_16},
```

Comment:
Is it necessary to escape the backspace here? (just for curiosity)

Reply:
good catch. apparently not, judging from the rest of the file